src/test/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionResolutionTest.java

```diff
@@ -2528,6 +2537,116 @@ public void extensionMetadata_isolatedDev() throws Exception {
         .isEqualTo("Updated use_repo calls for isolated usage 'ext2' of @ext//:defs.bzl%ext");
   }
 
+  @Test
+  public void facts_supportedTypes() throws Exception {
+    var result =
+        evaluateSimpleModuleExtension(
+            """
+            return ctx.extension_metadata(\
```

Comment:
why all these backslashes?

Reply:
The function requires the arg to be a single line or the indentation will be off.

Reply:
Didn't know this exists until now, updated